### PR TITLE
Time Method implementation

### PR DESCRIPTION
**Pull request by MehmetTerc**
I implementated the issue

### CODE REVIEW

**Comment by MehmetTerc** [comment]
It was the Question 33 and i make a small task to solve

File: README.md [hunk 1]
```diff
@@ -1065,4 +1065,27 @@ Thus, the final output of the code will be [ 11235, 3.14159, 'Centauri', 'canine
...
+const difference_in_time =us_election.getTime()-date1.getTime()
+const difference_in_days = difference_in_time  / (1000 * 3600 * 24);
+console.log(parseInt(difference_in_days,10)+ " Days")
+```
+
+</p>
+</details>
+=======
 ---
+
```
Comment (kirandash) [inline README.md:1091]: Please note that the questions are supposed to be MCQ. Can you please update the question with multiple choices that user can pick from.

Please have a look at other questions for inspiration

File: .idea/checkstyle-idea.xml
Comment (kirandash) [inline .idea/checkstyle-idea.xml:1]: Please add .idea to .gitignore file so that they are not committed to the repo.

**Review by kirandash — CHANGES_REQUESTED** [review]
Thanks for the PR! The question is nice but it doesn't meet the requirement of MCQ for this project

**Comment by MehmetTerc** [comment]
can you check it again

**Comment by kirandash** [comment]
> can you check it again

@MehmetTerc Thanks a lot for making the adjustments. 

I made tiny adjustments to the question and answer format but still using your content. 

Also, please note that when you have already pushed the files, and then you add it to gitignore, you will have to do some additional steps to remove it from remote repository. 

Adding the steps here for your reference: 

```
git rm -r --cached .
git add .
git commit -m "clean up pushed .idea files"
git push
```

I have already done this. But adding here for your reference. 

You can read more at: https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository

**Comment by kirandash** [comment]
fixes #23

**Pull request merged**